spec/unit/provider/package/pacman_spec.rb

```diff
@@ -10,6 +10,12 @@
   let(:resource) { Puppet::Type.type(:package).new(:name => 'package', :provider => 'pacman') }
   let(:provider) { described_class.new(resource) }
 
+  if Puppet.features.microsoft_windows?
```

Comment:
Looking at this in a little more detail, it looks like maybe `require 'English'` is sufficient to bring `$CHILD_STATUS` into scope on Windows.

I wonder if we can then add it to the top-level include that all specs drag in.... I believe that is `spec/spec_helper.rb`